profiles/schema.py

```diff
@@ -470,6 +495,8 @@ class Query(graphene.ObjectType):
     @staff_required(required_permission="view")
     def resolve_profile(self, info, **kwargs):
         service = Service.objects.get(service_type=kwargs["serviceType"])
+        # serviceType passed on to the sub resolvers
+        info.context.service_type = kwargs["serviceType"]
```

Comment:
looks ok, I think 😅

Reply:
I definitely prefer this to passing serviceType twice at different query levels 😅 🌞